### ISSUE TITLE
No more appImage builds since v1.1.1?

### ISSUE THREAD

**Issue by dandv** [kanryu/quickviewer#130]
[Releases](https://github.com/kanryu/quickviewer/releases) after v1.1.1 no longer include the .appImage file. Is Linux no longer supported?

**Comment by kanryu** [comment]
There is no Linux environment at hand. In the past, I was building because of technical interest, but I omit it now because it is very troublesome.

**Comment by dandv** [comment]
GitHub Actions provides a free Linux environment, and it can automatically build on push.

https://github.com/features/actions

**Comment by docop** [comment]
Indeed, other appimage can be quite usefull. You can deploy centos quite eaily on virtualbox in windows. Thanks in advance. great app.
